docs/get-started/network-information.mdx

```diff
@@ -6,10 +6,96 @@ import InlineCopy from '@site/src/components/InlineCopy';
 
 For current and historical status information for Etherlink, see https://status.etherlink.com.
 
+For every network, we provide two endpoints: a public RPC endpoint and a relay
+endpoint. The public RPC endpoint can be consumed by Etherlink users and
+decentralized applications to interact with the chain. The relay endpoint
+exposes a restricted RPC API with only `eth_sendRawTransaction` and
+`eth_blockNumber` available. The relay endpoint should be preferred when
+setting up an [Etherlink node](/network/evm-nodes).
```

Comment:
```suggestion
`eth_blockNumber` available. In most cases, use the relay endpoint when
setting up an [Etherlink node](/network/evm-nodes).
```

Reply:
Maybe we can drop “In most cases” even?

Reply:
Sure, unless there are cases where you would need to use the public endpoint. "Public endpoint" isn't a great name, though, because both endpoints are publicly accessible. Maybe "full RPC endpoint?"

Reply:
I’ve tried something, let me know what you think!

Reply:
Yeah, I think that's clear.